dockers/base-cuda/Dockerfile

```diff
@@ -14,7 +14,8 @@
 
 ARG CUDA_VERSION=11.3.1
 
-FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu20.04
+# TODO: Update OS to ubuntu20.04 when dropping CUDA 10.2
+FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu18.04
```

Comment:
maybe, lets add ubuntu version as argument

Reply:
Will address it when I get back home in a few hours!

Reply:
Sorry for the delay. Addressed!